apps/hash-frontend/src/pages/shared/entity-type-page/shared/entity-type-header.tsx

```diff
@@ -39,17 +39,13 @@ export const EntityTypeHeader = ({
   const [showExtendTypeModal, setShowExtendTypeModal] = useState(false);
 
   const isLatest =
-    !latestVersion || extractVersion(entityType.$id) === latestVersion;
-  const latestVersionUrl = entityType.$id.replace(/\d+$/, `${latestVersion}`);
+    !latestVersion || extractVersion(entityTypeSchema.$id) === latestVersion;
+  const latestVersionUrl = entityTypeSchema.$id.replace(
+    /\d+$/,
+    `${latestVersion}`,
+  );
```

Comment:
Nit: this works, but it might be slightly more readable if we were to use a combination of `extractBaseUrl` and `versionedUrlFromComponents` here. There is a type error this would introduce though with the usage of `latestVersion`.
```suggestion
  const latestVersionUrl = versionedUrlFromComponents(
    extractBaseUrl(entityTypeSchema.$id),
    latestVersion ?? 0,
  );
```

Reply:
Updated in https://github.com/hashintel/hash/pull/3984/commits/999f11786f44303ff3f57c81264a26f72890ffcd